superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/dndControls.tsx

```diff
@@ -244,8 +244,7 @@ export const dndGranularitySqlaControl: typeof dndSeriesControl = {
   default: (c: Control) => c.default,
   clearable: false,
   canDelete: false,
-  ghostButtonText: t('Drop temporal column here'),
-  clickEnabledGhostButtonText: t('Drop a temporal column here or click'),
+  ghostButtonText: t('Drop temporal column here or click'),
```

Comment:
Was leaving the "a" out intentional? I'm ok both ways but if we change the string existing translations will break.

Reply:
Good eye! Definitely not intentional